### DISCUSSION TITLE
Adguard doesnt start automatically after reboot

### DISCUSSION THREAD

**Discussion by NorthantsPete** [AdguardTeam/AdGuardHome#2832] [General]
I use CURL to install, and im using Tinycore as its small
I install and configure adguard ok but on reboot of the linux box Adguard is no more

How can we get it to run on boot?
Ive tried lots of distros all do the same thing so i cant be the first?

thanks

**Comment by ameshkov** [comment]
Well, what kind of service does AdGuard Home register? What does Tinycore use, Is it systemd or sysv?
